src/shared/CartButton/index.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { Link, LinkProps } from 'react-router-dom';
+import ShoppingCartIcon from '@material-ui/icons/ShoppingCart';
+import IconButton from '@material-ui/core/IconButton';
+
+// required for react-router-dom < 6.0.0

```

Comment:
Could we maybe update to >= 6.0.0 instead of doing this? Just a thought, no action required.